validator/project_validator_test.go

```diff
@@ -1501,7 +1511,8 @@ func TestCheckModules(t *testing.T) {
 					},
```

Comment:
nit: remove the `// should fail` comments above, and/or split the last module (with the SSH clone format) into a separate test case. That way, we can know that the warning comes from the SSH format and not some other reason.